bottypes/command_descriptor.py

```diff
@@ -3,6 +3,12 @@ class CommandDesc():
     def __init__(self, command, description, args, opt_args, is_admin_cmd=False):
         self.command = command
         self.description = description
-        self.arguments = args
-        self.opt_arguments = opt_args
+        if args == None:
+            self.arguments = []
```

Comment:
Can we simplify this to `self.arguments = args or []`?

Reply:
Done